apps/wallet/src/generated/station/station.did

```diff
@@ -1989,6 +2029,8 @@ service : (opt SystemInstall) -> {
   get_user_group : (input : GetUserGroupInput) -> (GetUserGroupResult) query;
   // List all user groups of the station.
   list_user_groups : (input : ListUserGroupsInput) -> (ListUserGroupsResult) query;
+  // Get canister status of a canister controlled by the station.
+  canister_status : (input : CanisterIdRecord) -> (CanisterStatusResult);
```

Comment:
The current format for inputs within our spec is to use `Input` as a suffix, for this case it would mean, `CaniterStatusInput`, and even though the existing api for canister status uses `CanisterIdRecord` i would assume using `CaniterStatusInput` as a name changes nothing and should still be compatible no?

Reply:
Indeed. Renaming type synonyms in the did file is harmless => done.